1apply-css/1styles.css

```diff
@@ -0,0 +1,21 @@
+   div {
+       color: purple;
+       font-family: sans-serif;
+       color: blue;
+       padding-bottom: 1em;
+   }
+
+   li{
+    color: purple
+   }
+
+
+   .highlight{
+    color: green;
+   }
```

Comment:
Look to be precise in your code stylistically. This code is technically without error, but there's extra whitespace to the left, inconsistent indent, a missing `;` in the `li` block, and inconsistent spaces before `{`. This kind of precision not only makes the code more legible for other developers, but will help you learn to be precise in other aspects of development as well.